Amethyst/Layout/Layout.swift

```diff
@@ -156,6 +156,11 @@ class ReflowOperation: Operation {
     }
 }
 
+// a reflow operation that organizes frames
+protocol FrameReflower {
+    func frameAssignments() -> [FrameAssignment]
```

Comment:
This could maybe just be on `ReflowOperation` and be optional for the sake of operations that don't have assignments (e.g., floating layout).

Reply:
I think the reason I put it here is to be able to share code between all layouts that are framed; `FramedLayout` being the glue that allows you to mark a layout as taking advantage of this code.  i.e. if you want to expose the assignments for "all reflow operations that come from a framed layout", I don't think `ReflowOperations` themselves know enough about their corresponding layout types to do this.